main.go

```diff
@@ -26,6 +26,7 @@ var Options struct {
 	HTTPSCertFile         string `envconfig:"HTTPS_CERT_FILE"`
 	HTTPSCAFile           string `envconfig:"HTTPS_CA_FILE"`
 	ListenPort            string `envconfig:"LISTEN_PORT" default:"8080"`
+	PlainHTTPListenPort   string `envconfig:"LISTEN_HTTP_PORT"`
```

Comment:
I'd make the variable and env var match.
So maybe `HTTPListenPort` and `HTTP_LISTEN_PORT`